server/routes/offenceRoutes.ts

```diff
@@ -786,6 +799,73 @@ export default class OffenceRoutes {
     )
   }
 
+  public getFineAmount: RequestHandler = async (req, res): Promise<void> => {
```

Comment:
There is a journey to get to this page via the edit offence screen, look in other pages to add a `submitToEditOffence` query to this page

Reply:
I've added this in

Reply:
`submitToEditOffence` needs to be added to the options of the page render too https://github.com/ministryofjustice/hmpps-remand-and-sentencing/blob/0d42b051dfd3afb099cc51dc11333b6f73e9cc7c/server/routes/offenceRoutes.ts#L825